README.md

```diff
@@ -3,155 +3,17 @@
 We use Tock to track our time. You can read more about Tock in this [blog post](https://18f.gsa.gov/2015/05/21/tockingtime/) about its features and the [handbook page](https://handbook.18f.gov/tock/) for guidance and instructions.
 
 [![CircleCI](https://circleci.com/gh/18F/tock.svg?style=svg)](https://circleci.com/gh/18F/tock)
+[![Dependency Status](https://gemnasium.com/badges/github.com/18F/tock.svg)](https://gemnasium.com/github.com/18F/tock)
 
-![Screenshot of Tock](https://github.com/18F/tock/blob/master/Screen%20Shot%202016-12-05%20at%2011.30.54%20AM.png)
+![Screenshot of Tock](https://github.com/18F/tock/blob/master/Screen%20Shot%202016-12-05%20at%2011.30.54%20AM.png?raw=true)
 
 If your team uses Tock and Slack, you might also find the ["angrytock" reminder bot](https://github.com/18F/angrytock) helpful.
 
-## Getting Started
+## Documentation
 
-1. Install [Docker][]. If you're on OS X, install Docker for Mac. If you're on Windows, install Docker for Windows.
+Read the project documentation in [the `/docs`
+directory][gh-docs]. Interested in the API docs? Read the API documentation in
+[the `/api-docs` directory][gh-api-docs].
```

Comment:
For some reason, this mixture of command and question feels awkward to me. My suggestion would be something more like "General project documentation for developers is in the /docs directory. API documentation is in /api-docs.

That also helps clarify that the "Documentation" heading is intended for devs, not end-users of the product.